casadm/cas_lib.c

```diff
@@ -2792,7 +2790,7 @@ int list_caches(unsigned int list_format, bool by_id_path)
 			"%s,%u,%s,%s,%s,%s\n",
 			"cache", /* type */
 			curr_cache->id, /* id */
-			curr_cache->device, /* device path */
+			curr_cache->standby_detached ? "detached" : curr_cache->device, /* device path */
```

Comment:
We may rather want to print "-" as a device name (like we do for detached cores exp objects) and change status (`tmp_status`) so it would print "Standby" for standby attached, and "Standby detached" for detached case.